net-libs/libtelnet/libtelnet-0.23.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils autotools
```

Comment:
Some background: eutils contained some things which ended up being included in various EAPIs.

So, sometimes you end up with some really common things that once needed it, and it ends up lingering around even though it's long-obsolete. Not needed here, and always good to keep an eye out for it!

(Also, we sort inherits).

Reply:
Thanks, it's removed